TaskSharper.DataAccessLayer.Google/Authentication/GoogleAuthentication.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Calendar.v3;
+using Google.Apis.Util.Store;
+
+namespace TaskSharper.DataAccessLayer.Google.Authentication
+{
+    public static class GoogleAuthentication
+    {
+        private static readonly string[] Scopes = { CalendarService.Scope.Calendar };
+        public static UserCredential Authenticate()
+        {
+            UserCredential credential;
+
+            using (var stream = new FileStream("client_secret.json", FileMode.Open, FileAccess.Read))
+            {
+                string credPath = Environment.GetFolderPath(Environment.SpecialFolder.Personal);
+                credPath = Path.Combine(credPath, ".credentials/calendar.json");
+
+                credential = GoogleWebAuthorizationBroker.AuthorizeAsync(
+                    GoogleClientSecrets.Load(stream).Secrets,
+                    Scopes,
+                    "user",
+                    CancellationToken.None,
+                    new FileDataStore(credPath, true)).Result;
+                //Console.WriteLine("Credential file saved to: " + credPath);
```

Comment:
Should this Console.writeline still be there?

Reply:
Done